### ISSUE TITLE
Minimap2: Paired ends option faulty

### ISSUE THREAD

**Issue by darkskiez** [galaxyproject/tools-iuc#2432]
Even when you choose paired end data files this is not passed to minimap2 correctly.

https://github.com/lh3/minimap2#map-short-accurate-genomic-reads

"Two reads are considered to be paired if they are adjacent in the
input stream and have the same name (with the /[0-9] suffix trimmed if
present)."

I had thought it was a file naming issue in galaxy with the dat file renaming, but I think from my understanding of the minimap code only the sr mapping mode supports paired ends, so this option should only be available when that mapping is selected if you can confirm my suspicions.

https://github.com/lh3/minimap2/blob/6762368cf0fdee289f757abb267e7ed43867ed52/options.c#L108

**Comment by mvdbeek** [comment]
That's for interleaved streams.
```
minimap2 -ax sr ref.fa read1.fq read2.fq > aln.sam     # paired-end alignment
```
Is what's implemented in the wrapper.

**Comment by darkskiez** [comment]
Yes I had accidentally selected the wrong mode (not sr) but could still select paired end data files, that should be blocked or highlighted for an improved user experience.

**Comment by mvdbeek** [comment]
fair enough

**Comment by bernt-matthias** [comment]
This seems to be implemented now, i.e. only `sr` is available for paired data.
